src/PostsViewHelper.php

```diff
@@ -19,11 +19,17 @@ public function displayAllPosts(array $posts)
         foreach ($posts as $post) {
 
             $postId = $post->id;
+
             $dislikes = $postModel->dislikeCount($postId);
             $likes = $postModel->likeCount($postId);
             $formattedDate = date('d/m/Y', strtotime($post->dateTime));
 
+            $controversial = '';
+            if ($dislikes['COUNT(`reaction`)'] > ($likes['COUNT(`reaction`)'] * 1.5)) {
```

Comment:
In your DB query that gets these counts, you should give them aliases so you don't have to repeat the SQL here